database/migrations/2017_04_27_141550_create_voucher_submission_pivot_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateVoucherSubmissionPivotTable extends Migration
```

Comment:
Could this be simplified? Do we really need many to many?

Is a submission a transaction that submits possible multiple vouchers AND a voucher can be used in more than one submission?